services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -1502,6 +1501,52 @@ specDeleteUser = do
       deleteUser_ (Just tok) (Just $ scimUserId storedUser) (env ^. teSpar)
         !!! assertTrue_ (inRange (200, 499) . statusCode)
 
+    context "user with scim, no saml, validated email, password" $ do
+      it "works" $ do
+        env <- ask
+        let brig = env ^. teBrig
+            spar = env ^. teSpar
+            galley = env ^. teGalley
+
+        (_, tid) <- call $ createUserWithTeam brig galley
+        tok <- registerScimToken tid Nothing
+
+        email <- randomEmail
+        scimUser <- randomScimUser <&> \u -> u {Scim.User.externalId = Just $ fromEmail email}
+        scimStoredUser <- createUser tok scimUser
+        let uid = scimUserId scimStoredUser
+
+        do
+          inv <- call $ getInvitation brig email
+          Just inviteeCode <- call $ getInvitationCode brig tid (inInvitation inv)
+          registerInvitation email (Name "Alice") inviteeCode True
+          call $ headInvitation404 brig email
+
+        deleteUser_ (Just tok) (Just uid) spar
+          !!! const 204 === statusCode
+        aFewTimes (getUser_ (Just tok) uid spar) ((== 404) . statusCode)
+          !!! const 404 === statusCode
+
+    context "user not touched via scim before" $ do
+      it "works" $ do
+        env <- ask
+        let brig = env ^. teBrig
+            spar = env ^. teSpar
+            galley = env ^. teGalley
+
+        (owner, tid) <- call $ createUserWithTeam brig galley
+        tok <- registerScimToken tid Nothing
+
+        memberInvited <- call (inviteAndRegisterUser brig owner tid)
+        let uid = userId memberInvited
```

Comment:
Not important enough for a ci re-run, but I can't help myself :)

```suggestion
        uid <- userId <$> call (inviteAndRegisterUser brig owner tid)
```

Reply:
i couldn't either ;)